plugin-examples/pom.xml

```diff
@@ -1,205 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
```

Comment:
If we add the parent as a dependency, all the parent pom's dependencies will be transitive dependencies of the plugin. This means that if we wanted to shade it with all its dependencies in a later stage we would re-add (with a different package name) many of Styx dependencies we don't really need...

Reply:
I would probably recommend using composition (i.e., importing a pom) here. In any case, it would be great if you could verify that if we try to use the add the plugin to Styx it's easy to include the required dependencies only.

Reply:
This change was added to ensure that the plugin examples build properly as part of the Styx project. I will see if any alternative is possible.